nselib/smb.lua

```diff
@@ -974,45 +976,37 @@ end
 --      * 'server_challenge' A random string used for challenge/response
 --      * 'domain'           The server's primary domain or workgroup
 --      * 'server'           The server's name
-function negotiate_protocol(smb, overrides)
+-- @param smb The SMB object associated with the connection.
+-- @param overrides Overrides table.
+-- @return (status, dialect) If status is true, the negotiated dialect in human readable form is returned as the second value.
```

Comment:
NSEdoc supports multiple `@return` statements. The convention for this sort of thing is:

```lua
-- @return Boolean status
-- @return The negotiated dialect in human-readable form, or an error message if status is false
```